.rubocop.yml

```diff
@@ -330,6 +330,7 @@ Style/FrozenStringLiteralComment:
 
 Style/HashSyntax:
   Enabled: true
+  EnforcedShorthandSyntax: never
```

Comment:
After changing the target Ruby version to 3.1, it would be possible to use the hash shorthand syntax

```rb
# bad
{foo: foo, bar: bar}

# good
{foo:, bar:}
```

however, there are 62 offenses across several files and I didn't want to have a lot of changes in this PR

Also, I personally do not like this syntax but I'm not against it

Reply:
Same here. I think that's best. We'll let that happen naturally over time if others choose as newer versions become the default. Thank you.